== TEST (2019-05-15 15:43:07) 
-- CODE:
import math

x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 15:46:06) 
-- CODE:
import math

x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 15:46:37) 
-- CODE:
import math

x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 15:50:35) 
-- CODE:
import math

x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-15 15:51:01) 
-- CODE:
import math

x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- EXECUTION TIME:
0.145448
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1728/2505/main.py", line 3, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 15:53:09) 
-- CODE:
import math

x = eval()
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval()
TypeError: eval expected at least 1 arguments, got 0
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 15:53:30) 
-- CODE:
import math

x = eval('pi/4 2')
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval('pi/4 2')
  File "<string>", line 1
    pi/4 2
         ^
SyntaxError: unexpected EOF while parsing
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 15:53:43) 
-- CODE:
import math

x = eval('pi/4')
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval('pi/4')
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 15:54:00) 
-- CODE:
import math

x = eval("pi/4")
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval("pi/4")
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 15:55:10) 
-- CODE:
import math

x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 15:55:59) 
-- CODE:
import math

x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 16:03:16) 
-- CODE:
import math

x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,8))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-15 16:04:54) 
-- CODE:
import math

x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,10))
-- EXECUTION TIME:
0.225824
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1728/2505/main.py", line 3, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-15 16:05:43) 
-- CODE:
import math
x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,10))
-- EXECUTION TIME:
0.180843
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1728/2505/main.py", line 2, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 16:17:18) 
-- CODE:
import math
x = eval(input(' '))
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 16:17:45) 
-- CODE:
import math
x = eval(input('pi'))
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 16:18:02) 
-- CODE:
import math
x = eval(input('pi'))
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 16:19:27) 
-- CODE:
import math
x = eval()
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    x = eval()
TypeError: eval expected at least 1 arguments, got 0
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-15 16:19:46) 
-- CODE:
import math
x = eval(input())
n = int(input())

s = 0
i = 0

while i<n:
    s = ((x ** (2 * i + 1)) * ((-1) ** (i))) / (math.factorial(2 * i + 1)) + s
    i=i+1

print(round(s,10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    x = eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
